lua/dap-disasm.lua

```diff
@@ -144,14 +149,14 @@ write_buf = function(pc)
     end
 
     local line = " "
-    if fmts.address then
-      line = line .. string.format(fmts.address .. ":\t", ins.address)
+    if columns.address then
+      line = line .. string.format("%-" .. max_len.address .. "s:\t", ins.address)
```

Comment:
Isn't the format string resulting from `"%-" .. max_len.address .. "s:\t"` the same as before?